apps/aecore/lib/aecore/structures/spend_tx.ex

```diff
@@ -134,5 +141,25 @@ defmodule Aecore.Structures.SpendTx do
     Map.put(account_state, :balance, new_balance)
   end
 
+  @spec is_minimum_fee_met?(SignedTx.t(), :miner | :pool | :validation) :: boolean()
+  def is_minimum_fee_met?(tx, identifier) do
```

Comment:
What is the function of the identifier. What is the difference between the `:pool` and the `:miner` identifiers

Reply:
Spend tx minimum fee is calculated by a formula in which one of the constants can be set by the miner meaning that the calculation can be different in the 2 cases